src/main/java/graphql/execution/ExecutionContext.java

```diff
@@ -170,6 +168,9 @@ public ValueUnboxer getValueUnboxer() {
         return valueUnboxer;
     }
 
+    @ExperimentalApi
+    public boolean propagateErrors() { return propagateErrors; }
```

Comment:
Nice use of @ExperimentalApi - can we have a javadoc comment in it please

Reply:
✅ let me know what you think